packages/mocks/package.json

```diff
@@ -65,7 +65,7 @@
     "@web/storybook-prebuilt": "^0.1.37",
     "@web/storybook-utils": "^1.0.0",
     "lit": "^2.7.5",
-    "msw": "^0.0.0-fetch.rc-23"
```

Comment:
I was always thinking that in this case it will be practically pinned even with the caret
thinking so based on recent experience testing storybook 7 prereleases where new rc's were never automatically picked up

Reply:
from a chat with Pascal

> somehow installing msw@^0.0.0-fetch.rc-23 results in 0.0.0-fetch.rc-9 to be installed instead o.O

apparently the suffix is interpreted as a string

Reply:
yeah I thought the same thing. We did have the version locked on the previous rc version, we used to have: `"0.0.0-fetch.rc-15"` for a long time. Silly mistake to have the `^` in there